HIPs/solang.md

```diff
@@ -0,0 +1,260 @@
+and call various contract functions.
+
+# Effort and Resources
+
+Initial funding for Solang provided was through two grants from the Web3
+Foundation and another grant from the Solana Foundation.
+
+Currently there are three people working on Solang full time:
+
+Solana Labs:
+- Sean Young [<sean@mess.org>](mailto:sean@mess.org)
+- Lucas Steuernagel [<lucas.tnagel@gmail.com>](mailto:lucas.tnagel@gmail.com)
+
+Parity Tech:
+- Cyrill Leutwiler [<bigcyrill@hotmail.com>](mailto:bigcyrill@hotmail.com)
+
+It is expected that the integration with Solana will finish end by the end of 2022,
+attracting more users and more relevance for the project. As the compatibility with
+both Parity and Solana matures, the development focus is supposedly shifting to
+performance improvements and new features to the Solidity language.
+
+The Web3 Foundation funds gitcoin bounties for
+[certain issues](https://github.com/hyperledger-labs/solang/issues?q=is%3Aissue++label%3Abounty-M%2Cbounty-S%2Cbounty-L). These are small bounties (about 400US$) for small issues.
+
+# How To
+
+The source code is hosted on [github](https://github.com/hyperledger-labs/Solang)
+and the [documentation](https://Solang.readthedocs.io/en/latest/) is extensive.
+There is documentation on how to [run Solang on the command line](https://Solang.readthedocs.io/en/latest/running.html),
+and blockchain specific instructions for [Solana](https://solang.readthedocs.io/en/latest/targets/solana.html),
+[Substrate](https://solang.readthedocs.io/en/latest/targets/substrate.html), and
+[Burrow](https://solang.readthedocs.io/en/latest/targets/burrow.html).
+
+# References
+
+N/A
+
+# How does Hyperledger Solang meet the incubation entry considerations?
+### Codebase
+* _Code should exist as open source software in some form. Previous accepted projects have come up through labs (e.g., Cactus, Ursa); while others previously had stand alone governance prior to joining (e.g., Besu)._
+
+  **Solang has been in Hyperledger Labs since December, 2018. This is where the project started.**
+
+* _DCO sign off should exists in the code repository. If not 100% ready, the code must be capable of becoming compliant upon entry (i.e., squash commit)._
+
+  **Solang has required DCO signoffs since entry into the Labs**
+
+### Maintainers
+* _The project should have multiple maintainers. These maintainers need not be from different companies; however, having maintainers from different companies is seen as a positive sign. Proposals with only one maintainer have been rejected by prior TSCs._
+
+  **The maintainers can be found listed in [MAINTAINERS.md](https://github.com/hyperledger-labs/solang/blob/main/MAINTAINERS.md). Multiple companies are represented in the list.**
+
+* _The project should have demonstrable examples of POC/production uses publicly available._
+
+  **Solang is being used for multiple engagements, none of them public yet. One engagement is aiming for production in the October 2022.**
```

Comment:
Are you able to share more details in a generalized form? for instance are these engagements in the Solana or PolkaDot communities, are they used in permissioned chains or public chains, etc? any from enterprise use cases or only web3 startups? It'll help a lot in getting a sense of how Solang is helping the Hyperledger mission.

Reply:
Good idea, I've add some words